### PR TITLE
IT-21 Dynamic Properties Visible After 5 Seconds Button test finished

### PR DESCRIPTION
**Pull request by 1Serhiiqa**
IT-21 is ready to final check

### CODE REVIEW

File: Alifanov.ITLeadersCourse/NUnitFramework/Pages/Elements/DynamicPropertiesPage.cs [hunk 1]
```diff
@@ -19,5 +21,19 @@ public bool IsColorButtonColorChanged()
...
+        public bool IsVisibleAfterButtonEnabled()
+        {
+            Waiter.WaitUntilElementToBePresent(VisibleAfterButtonLocator);
+
+            return Waiter.WaitUntilElementToBeEnabled(VisibleAfterButton);
+        }
+
+        public bool IsVisibleAfterButtonClickable()
+        {
+            VisibleAfterButton.Click();
```
Comment (user-yalif) [inline Alifanov.ITLeadersCourse/NUnitFramework/Pages/Elements/DynamicPropertiesPage.cs:34]: Why do you need this click here?

File: Alifanov.ITLeadersCourse/NUnitFramework/NUnitFramework.csproj [hunk 2]
```diff
@@ -24,4 +24,8 @@
     </None>
   </ItemGroup>
 
+  <ItemGroup>
```
Comment (user-yalif) [inline Alifanov.ITLeadersCourse/NUnitFramework/NUnitFramework.csproj:27]: This ItemGroup section should be removed from this file

File: Alifanov.ITLeadersCourse/NUnitFramework/Pages/Elements/DynamicPropertiesPage.cs [hunk 3]
```diff
@@ -1,15 +1,17 @@
 ﻿namespace NUnitFramework.Pages.Elements
 {
     using NUnitFramework.Helpers;
-    using NUnitFramework.Logging;
     using OpenQA.Selenium;
 
     public class DynamicPropertiesPage : BasePage
     {
+        private By VisibleAfterButtonLocator = By.CssSelector("button#visibleAfter");
         private IWebElement EnableAfterButton => FindElement(By.CssSelector("button#enableAfter"));
```
[the remark this answers is not in the document]
Comment (1Serhiiqa) [reply Alifanov.ITLeadersCourse/NUnitFramework/Pages/Elements/DynamicPropertiesPage.cs:9]: done

File: Alifanov.ITLeadersCourse/NUnitFramework/NUnitFramework.csproj (hunk 2 above)
Comment (1Serhiiqa) [reply Alifanov.ITLeadersCourse/NUnitFramework/NUnitFramework.csproj:27]: done

File: Alifanov.ITLeadersCourse/NUnitFramework/Pages/Elements/DynamicPropertiesPage.cs (hunk 1 above)
Comment (1Serhiiqa) [reply Alifanov.ITLeadersCourse/NUnitFramework/Pages/Elements/DynamicPropertiesPage.cs:34]: I guess there are two places when this click can describe - this file or a file with tests. I like this place) (I took as an example ButtonsPage.cs file.)

Comment (user-yalif) [reply Alifanov.ITLeadersCourse/NUnitFramework/Pages/Elements/DynamicPropertiesPage.cs:35]: Remove it, please

**Review by user-yalif — APPROVED** [review]

**Pull request merged**